Upgrade Django to v5.0 for the errorreports service (#62)
mantidproject/errorreports: DevelopmentSetup.md

* the port to run on
* the name of the database user
* the database password

> * the trusted CSRF origin URLs as appropriate. For developer testing set either http://localhost:8083 or https://errorreports.a.staging-mantidproject.stfc.ac.uk if testing on staging environment. But for the production usage only set this to https://errorreports.mantidproject.org

Reviewer: Just think this is a bit clearer:
```suggestion
* the trusted CSRF origin URLs (`DJANGO_CSRF_TRUSTED_ORIGINS`):

   * For local developer testing: `http://localhost:8083`
   * For the staging environment: `https://errorreports.a.staging-mantidproject.stfc.ac.uk`
   * For production: `https://errorreports.mantidproject.org`
```
I think it would also be good to be more specific about what names the variables here are called in the `.env` file. (E.g: Is "the port to run on" the `HOST_PORT` or `DB_PORT`?)
